nixos/modules/services/misc/domoticz.nix

```diff
@@ -0,0 +1,89 @@
+{ lib, pkgs, config, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.domoticz;
+  pkgDesc = "Domoticz home automation";
+
+in {
+
+  options = {
+
+    services.domoticz = {
+      enable = mkEnableOption pkgDesc;
+
+      user = mkOption {
```

Comment:
Can we use systemd dynamic users instead of specifying the user/group? http://0pointer.net/blog/dynamic-users-with-systemd.html

Reply:
https://github.com/nuxeh/nixpkgs/pull/1/files made a PR against your branch that changes it to use StateDirectory/DynamicUser check it out!

Reply:
Interesting, hadn't heard about this, and previously thought having a configuration option for state dir was a bit clunky.

But thanks, merged.